braindecode/models/eegitnet.py

```diff
@@ -0,0 +1,234 @@
+# Authors: Ghaith Bouallegue <ghaithbouallegue@gmail.com>
+#
+# License: BSD-3
+import torch
+from torch import nn
+
+from .modules import Ensure4d, Expression
+
+
+class _DepthwiseConv2d(torch.nn.Conv2d):
+    def __init__(
+            self,
+            in_channels,
+            depth_multiplier=2,
+            kernel_size=3,
+            stride=1,
+            padding=0,
+            dilation=1,
+            bias=True,
+            padding_mode="zeros",
+    ):
+        out_channels = in_channels * depth_multiplier
+        super().__init__(
+            in_channels=in_channels,
+            out_channels=out_channels,
+            kernel_size=kernel_size,
+            stride=stride,
+            padding=padding,
+            dilation=dilation,
+            groups=in_channels,
+            bias=bias,
+            padding_mode=padding_mode,
+        )
+
+
+
+class _InceptionBlock(nn.Module):
+    def __init__(self, in_ch, out_ch, kernel_sz, in_channels):
+        super().__init__()
+        self.inception_block = nn.Sequential(
+            *[
+                nn.Conv2d(
+                    in_ch, out_ch, kernel_size=(1, kernel_sz), padding="same", bias=False
+                ),
+                nn.BatchNorm2d(out_ch),
+                _DepthwiseConv2d(
+                    out_ch,
+                    kernel_size=(in_channels, 1),
+                    depth_multiplier=1,
+                    bias=False,
+                    padding="valid",
+                ),
+                nn.BatchNorm2d(out_ch),
+                nn.ELU(),
+            ]
+        )
+
+    def forward(self, x):
+      return self.inception_block(x)
```

Comment:
My idea of `_InceptionBlock` code:
```suggestion
class _InceptionBlock(nn.Module):
    def __init__(self, branches):
        super().__init__()
        self.branches = nn.ModuleList(branches)

    def forward(self, x):
        return torch.cat([branch(x) for branch in self.branches], 1)
```
It is quite generic, then creating modules would stay inside the main class `EEG_ITNet`